essential-projects/messagebus_http
# 📦  use messagebusService to create message

## What did you change?

I replaced the creation of the message created through `POST /messagebus/publish`, so the message is now created thorugh the messagebusService, instead of creating a new instance of a Message class

## How can others test the changes?

given this package is registered to the ioc, you can try to `POST` to `/messagbus/publish` and see, if it still works. 

## PR-Checklist

Please check the boxes in this list after submitting your PR:
